pkg/kubectl-argo-rollouts/info/experiment_info.go

```diff
@@ -31,7 +31,7 @@ func NewExperimentInfo(
 		Message: exp.Status.Message,
 	}
 	expInfo.Icon = analysisIcon(exp.Status.Phase)
-	expInfo.Revision = int32(parseRevision(exp.ObjectMeta.Annotations))
+	expInfo.Revision = parseRevision(exp.ObjectMeta.Annotations)
```

Comment:
why do we need this change?

Reply:
The function was not checking the lower and upperbound limits of revision. Before the function was not able to handle if the revision exceeded the int32 upper bound limit. It was raising alerts on this.  I was not able to use int64 as javascript can't handle int64 datatypes. My idea was once it reaches the upperlimit, to make the revision start from 0. I am not sure abt this change, but let me know what you think abt this. I think we need to reset the revision to start back from 0.